optimizely/odp/odp_manager.py

```diff
@@ -78,7 +86,7 @@ def identify_user(self, user_id: str) -> None:
             self.logger.debug('ODP identify event is not dispatched (ODP disabled).')
             return
         if self.odp_config.odp_state() == OdpConfigState.NOT_INTEGRATED:
-            self.logger.debug('ODP Identify event is not dispatched (ODP not integrated).')
+            self.logger.debug('ODP identify event is not dispatched (ODP not integrated).')
```

Comment:
Line 107-114, send_event raises multiple exceptions. They should be processed separately at the top level. We can discuss again when integrated.